doc/source/response.rst

```diff
@@ -70,6 +70,9 @@ Dynamic Response
 ================
 
 A callback can be provided in place of any of the body elements.
+raw attribute also accepts callable that returns HTTPResponse.
+The HTTPResponse should have `preload_content=False` or it may not work properly.
```

Comment:
does it make sense to just always set `raw.preload_content = False` on the raw object? I forget what would happen if this isn't set?

Reply:
`enforce_content_length = False` should probably always be set as well?

Reply:
I think we leave them as is to avoid breaking backward compatibility.

Reply:
ok, so i can't set `preload_content` as that is acted upon in the constructor. Setting it afterwards has no impact.

For `enforce_content_length` i think it should probably be false - but the whole point of using `raw` is that the user provides this `HTTPResponse` object so i guess we shouldn't do anything that changes the values they set.